Makefile

```diff
@@ -112,8 +111,13 @@ kind-load-test-artifacts: $(KIND) ## Load the e2e testdata container images into
 
 ##@ Build
 
-BUILDCMD = sh -c 'mkdir -p $(BUILDBIN) && $(GORELEASER) build ${GORELEASER_ARGS} --single-target -o $(BUILDBIN)/manager'
-BUILDDEPS = manifests generate fmt vet $(GORELEASER)
+export GO_BUILD_ASMFLAGS ?= all=-trimpath=${PWD}
```

Comment:
Agree with @stevekuznetsov that it would be best to separate the dependencies and build into two targets.

Reply:
Ok, that seems reasonable. :+1: